Hax/css/screen.css

```diff
@@ -328,6 +328,15 @@ p code {
   color: #fff;
 }
 
+.branding .logo-askew {
+  animation-name: normal;
```

Comment:
Probably, though I'm not sure I've seen one that does it (but I also haven't really looked).